### DISCUSSION TITLE
'Enter' behavior inconsistent with vim

### DISCUSSION THREAD

**Discussion by geeksmith** [NeoVintageous/NeoVintageous#1033] [General]
Is there a reason that hitting 'Enter' doesn't move the cursor to the beginning of the line, as it does in vim? Is there an option to enable this behavior?

**Comment by geeksmith** [comment]
I also noticed that if the last line of the file is a blank line, 'Ctrl-G' goes to the line above the last line instead of the real end of the file.

**Comment by gerardroche** [comment]
This was actually fixed just recently: https://github.com/NeoVintageous/NeoVintageous/commit/af7283f9176a29a17251c3c729710ee6e6e2684a
Will be in the next patch release.

Ctrl+G ???

Do you have ctrl keys enabled? If you want to jump to a line you can use `{count}G`:

To jump to line 42 press 42G

**Comment by geeksmith** [comment]
Must have been a rough day -- 'Shift-G' is what I meant to say :-)

Thank you for the quick response, sorry for my slow one...
